lib/public/views/QcFlags/details/QcFlagDetailsPage.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+import { LabelPanelHeaderComponent } from '../../../components/common/panel/LabelPanelHeaderComponent.js';
+import { PanelComponent } from '../../../components/common/panel/PanelComponent.js';
+import { markdownDisplay } from '../../../components/common/markdown/markdown.js';
+import { qcFlagDetailsConfiguration } from './detailsConfiguration.js';
+import errorAlert from '../../../components/common/errorAlert.js';
+import { deleteButton } from '../../../components/common/form/deleteButton.js';
+import { detailsList } from '../../../components/Detail/detailsList.js';
+import spinner from '../../../components/common/spinner.js';
+
+/**
+ * Create panel with QC flag details display
+ *
+ * @param {QcFlagDetailsModel} detailsModel model
+ * @return {Component} panel
+ */
+const qcFlagDetailsComponent = (detailsModel) => {
+    const { qcFlag: remoteQcFlag } = detailsModel;
+
+    const commentPanel =
+    h(
+        PanelComponent,
+        [
+            remoteQcFlag.match({
+                Success: (qcFlag) => qcFlag.comment
+                    ? [
+                        h(LabelPanelHeaderComponent, 'Comment'),
+                        markdownDisplay(
+                            qcFlag.comment,
+                            {
+                                classes: 'w-100',
+                            },
```

Comment:
When you have only one prop and short lines, you can put it on one line

Reply:
removed